doc/api/fs.md

```diff
@@ -85,7 +85,7 @@ variable:
 Omitting the callback function on asynchronous fs functions is deprecated and
 may result in an error being thrown in the future.
 
-```txt
+```bash
```

Comment:
It seems `console` is what we use in other cases so I would use it instead of `bash`.